front/src/modules/relation-picker/components/SingleEntitySelect.tsx

```diff
@@ -16,6 +17,8 @@ import { isDefined } from '@/utils/type-guards/isDefined';
 
 import { useEntitySelectLogic } from '../hooks/useEntitySelectLogic';
 
+import 'react-loading-skeleton/dist/skeleton.css';
```

Comment:
I would import it in src/index.tsx as skeletons is a global feature and not tied to the SingleEntitySelect